src/snowcli/cli/nativeapp/policy.py

```diff
@@ -0,0 +1,25 @@
+import typer
+
+
+class PolicyBase:
+    exit_code: int = 0
+
+    def should_proceed(self, user_prompt: str):
+        pass
```

Comment:
ABC could be used to ensure interface implementation?

```suggestion
class PolicyBase(abc.ABC):
    exit_code: int = 0

    def should_proceed(self, user_prompt: str) -> bool:
        pass
```

Reply:
Sure thing, fixing in the upcoming commit. Thanks!